yukyungdeok/9325.py

```diff
@@ -0,0 +1,11 @@
+# 9325번 얼마?
+n = int(input())
+
+for i in range(n):
```

Comment:
```suggestion
for i in range(int(input()):
```
같은 식으로 하면 줄 좀 더 줄일 수 있어